tasmota/tasmota_xdrv_driver/xdrv_90_esp32_dingtian_relay.ino

```diff
@@ -70,7 +73,12 @@ uint32_t DingtianReadWrite(uint32_t outputs)
   }
   // ending
   digitalWrite(Dingtian->pin_rck, 1);    // rclk pulse to load '595 into output registers
-  digitalWrite(Dingtian->pin_pl, 0);      // re-enable '595 ouputs
+  if (PinUsed(GPIO_DINGTIAN_PL)) digitalWrite(Dingtian->pin_pl, 0);      // re-enable '595 ouputs (old board version)
+  if (PinUsed(GPIO_DINGTIAN_OE))
+  {
+    digitalWrite(Dingtian->pin_oe, 0);      // enable '595 ouputs (new board version)
+    DINGTIAN_SET_OUTPUT(Dingtian->pin_oe, 0);
+  }
```

Comment:
1) better test pin_oe than making a call to PinUsed() which is longer (insure that pin_oe has a default value meaning not used)
2) any way to avoid doing that at each loop but only once ?

Reply:
1. Ok
2. Yes I can change that